Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pre-commit = "==1.13.0"
+
+[packages]
+selenium = "==3.141.0"
+astroid = "==1.6.5"
```

Comment:
should we differentiate its version based on py3/py2?

Reply:
No, 1.6.5 is enough I think.
`astroid` is used by pylint. Pylint 1.9.3 is necessary to run it for both versions.
Once we drop Python 2.7, we can remove this deps and update the pylint, too.

Reply:
Then it makes sense to keep a TODO comment there